terraform/modules/concourse_worker/security_groups.tf

```diff
@@ -7,22 +7,23 @@ resource "aws_security_group" "worker" {
   }
 }
 
-resource "aws_security_group_rule" "worker_all_out" {
-  description       = "worker_all_out"
-  from_port         = 0
+#Currently required to get to docker hub. Need to fix this later.
```

Comment:
I've tested and this is required.
The reason appears to be that the workers are not configured to use the web proxy